source/includes/steps-enable-authentication-in-shardcluster-nodowntime-uac.yaml

```diff
@@ -0,0 +1,113 @@
+stepnum: 1
+title: Create the adminstrator user.
+level: 4
+ref: add-user-useradmin
+pre: |
+
+  Clients performing maintenance operations or user administrative
+  operations on the {{CLUSTER}} must authenticate as this user
+  at the completion of this tutorial. Create this user now to ensure
+  that you have seamless access to the cluster after enforcing authentication.
+
+  Create a user with the :method:`db.createUser()` method and assign it the
+  following roles:
+    
+  - :authrole:`clusterAdmin` on the ``admin`` database
+  - :authrole:`userAdmin` roles on the ``admin`` database
+
+action:
+  language: javascript
+  code: |
+    admin = db.getSiblingDB("admin")
+    admin.createUser(
+      {
+        user: "admin",
+        pwd: "<password>",
+        roles: [ 
+          { role: "clusterAdmin", db: "admin" },
+          { role: "userAdmin", db: "admin" } 
+        ]
+      }
+    )
+
+post: |
+
+  .. important::
+    
+     Passwords should be random, long, and complex to ensure system security
+     and to prevent or delay malicious access.
+     
+     While this tutorial uses a single administrative user, you should
+     consider applying the `principle of least privilege
+     <https://www.us-cert.gov/bsi/articles/knowledge/principles/least-privilege>`_
+     when configuring MongoDB users in your deployment.
```

Comment:
Not sure why using a single admin user would be counter to applying the principle of least privilege.  Because this sentence reads "While we do this, you should consider this alternative"